diagnostic.md

```diff
@@ -5,69 +5,75 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+this is where all the data is stored to and accessed from
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+we had already done that with the html, css, and javascript
 ```
 
 What does C.R.U.D stand for?
 
 ```md
-// your response here
+Create
+Read
+Update
+Delete
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+controller
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
-// your response here
+index, show, create, update, destroy
 ```
 
 A user action fires a `GET` request for `/people/1`. Explain in detail each step
 required for data to be returned to the client. (bullet points or ordered list)
 
 ```md
-// your response here
+1. the router issues a request to the controller for /people/1
+2. the controller issues a request to the model for get data for /people/1
+3. model tries to find the data associated for GET /people/1 and returns a response
+4. the controller recieves the response from the model and sends the response message to the viewer
 ```
 
 What is the command to generate a new rails-api app?
 
 ```bash
-// your response here
+bin/rails generate controller new-app-name
```

Comment:
That would only generate a controller not a new app. For this we can use:

`rails new <app_name>`